symfony.lock

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
After installation this lockfile already had changes. Can we fix that?
(symfony/symfony and symfony/polyfill-ctype were added).

Reply:
I added the updated symfony.lock